crates/errors/src/error_codes/META0012.md

```diff
@@ -0,0 +1,9 @@
+## META0012
+
+Trying to register a service endpoint whose supported service protocol versions is incompatible with the server. This indicates that you have to update your server to make it work together with the deployed SDK.
+
+Suggestions:
+
+* Check the compatibility matrix between SDK and server versions
```

Comment:
We could definitely use a page in the documentation for that.

Reply:
Once we have it, we can link it from this error.